src/adapter/src/coord/id_bundle.rs

```diff
@@ -118,23 +117,17 @@ impl CollectionIdBundle {
             .copied()
             .chain(self.compute_ids.values().flat_map(BTreeSet::iter).copied())
     }
-}
 
-impl Coordinator {
-    /// Resolves the full name from the corresponding catalog entry for each item in `id_bundle`.
+    /// Resolves the full name from the corresponding catalog entry for each item in this bundle.
     /// If an item in the bundle does not exist in the catalog, it's not included in the result.
-    pub fn resolve_collection_id_bundle_names(
-        &self,
-        session: &Session,
-        id_bundle: &CollectionIdBundle,
-    ) -> Vec<String> {
-        let mut names: Vec<_> = id_bundle
+    pub fn resolve_names(&self, catalog: &Catalog, conn_id: &ConnectionId) -> Vec<String> {
+        let mut names: Vec<_> = self
```

Comment:
I've moved `resolve_collection_id_bundle_names` to be a method on `CollectionIdBundle` instead of `Coordinator`, and rename it to `resolve_names`. This is to be able to call it from both the old and the new sequencing.